pkg/signup/service/signup_service.go

```diff
@@ -442,6 +442,16 @@ func (s *ServiceImpl) DoGetSignup(ctx *gin.Context, cl namespaced.Client, userna
 		signupResponse.DefaultUserNamespace = defaultNamespace
 	}
 
+	if defaultNamespace == "" {
+		// default namespace is not set, so we need to set the status to not ready
+		signupResponse.Status = signup.Status{
+			Ready:                false,
+			Reason:               toolchainv1alpha1.UserSignupNoDefaultNamespaceReason,
+			Message:              "No default namespace found",
```

Comment:
WDYT about this message instead?
```suggestion
			Message:              "Default namespace not available (yet)",
```